homeassistant/components/solaredge_local/manifest.json

```diff
@@ -4,5 +4,5 @@
     "documentation": "",
     "dependencies": [],
     "codeowners": ["@drobtravels"],
```

Comment:
Thank you for your contribution thus far! 🎖 Since this is a significant contribution, we would appreciate you'd added yourself to the list of code owners for this integration. ❤️ 

For more information about "code owners", see: [Architecture Decision Record 0008: Code owners](https://github.com/home-assistant/architecture/blob/master/adr/0008-code-owners.md).